test/verify/check-machines-dbus

```diff
@@ -484,8 +484,8 @@ class TestMachinesDBus(machineslib.TestMachines):
         args = self.startVm("subVmTest1")
 
         m.execute("virsh attach-disk --domain subVmTest1 --source /mnt/vm_one/mydiskofpoolone_1 --target vdd --targetbus virtio")
-        m.execute("virsh attach-disk --domain subVmTest1 --source /mnt/vm_one/mydiskofpoolone_2 --target vde --targetbus virtio --persistent")
-        m.execute("virsh attach-disk --domain subVmTest1 --source /mnt/vm_one/mydiskofpoolone_3 --target vdf --targetbus virtio --persistent")
+        m.execute("virsh attach-disk --domain subVmTest1 --source /mnt/vm_one/mydiskofpoolone_2 --target sdb --targetbus scsi --persistent")
+        m.execute("virsh attach-disk --domain subVmTest1 --source /mnt/vm_one/mydiskofpoolone_3 --target vde --targetbus virtio --persistent")
```

Comment:
I'm not sure how this device rename is related to this change? The test doesn't actually seem to change the removal of disks, or does it?

Reply:
It's just according to libvirt's docs, disks with target bus of scsi, sata, usb... should have target in format of "sdX".
And bugzilla is appears only for disks with target "sdX".
And then I moved the vdf -> vde to just keep it in aphabetic order, so the unique target naming is not all over the place/